src/Behat/Cores/Drupal7.php

```diff
@@ -292,4 +292,18 @@ public function formatString($string, array $params) {
     return format_string($string, $params);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function stateGet($key) {
```

Comment:
This methods need to be added to the interface.

Reply:
It seems I forgot to commit the CoreInterface modifications :(.

Thanks!